lib/content-services/src/lib/content-node-selector/content-node-selector.service.ts

```diff
@@ -67,15 +49,14 @@ export class ContentNodeSelectorService {
                 skipCount: skipCount
             },
             filterQueries: [
-                { query: `TYPE:'cm:folder'${ showFiles ? " OR TYPE:'cm:content'" : '' }` },
+                { query: `TYPE:'cm:folder'${showFiles ? " OR TYPE:'cm:content'" : ''}` },
                 { query: 'NOT cm:creator:System' },
                 ...parentFiltering
             ],
             scope: {
-                locations: ['nodes']
+                locations: 'nodes'
```

Comment:
why it's not an array?

Reply:
https://github.com/Alfresco/alfresco-js-api/blob/develop/src/api/search-rest-api/model/requestScope.ts#L37 should be a string I think not an array

Reply:
thanks for clarifying